packages/tools/src/utilities/voi/windowlevel/getLuminance.ts

```diff
@@ -0,0 +1,33 @@
+function getLuminance(imageData, x, y, width, height) {
+  x = Math.round(x);
+  y = Math.round(y);
+  const luminance = [];
+  let index = 0;
+  const pixelData = imageData.scalarData;
+  let spIndex, row, column;
+
+  if (imageData.color) {
+    for (row = 0; row < height; row++) {
+      for (column = 0; column < width; column++) {
+        spIndex = ((row + y) * imageData.columns + (column + x)) * 4;
+        const red = pixelData[spIndex];
+        const green = pixelData[spIndex + 1];
+        const blue = pixelData[spIndex + 2];
+
+        luminance[index++] = 0.2126 * red + 0.7152 * green + 0.0722 * blue;
+      }
+    }
+  } else {
+    for (row = 0; row < height; row++) {
+      for (column = 0; column < width; column++) {
+        spIndex = (row + y) * imageData.columns + (column + x);
+        luminance[index++] =
+          pixelData[spIndex] * imageData.slope + imageData.intercept;
```

Comment:
pixelData has slop and intercept already applied, i'm not sure why we are re-applying here